### ISSUE TITLE
text_recognition, I export my trained model to OpenVINO™ Intermediate Representation (IR) ,some errors occur!

### ISSUE THREAD

**Issue by jiazhaozhu** [opencv/openvino_training_extensions#297]
[ ERROR ]  List of operations that cannot be converted to Inference Engine IR:
[ ERROR ]      FusedBatchNormV3 (8)
[ ERROR ]          shadow/BatchNorm/FusedBatchNormV3
[ ERROR ]          shadow/BatchNorm_1/FusedBatchNormV3
[ ERROR ]          shadow/BatchNorm_2/FusedBatchNormV3
[ ERROR ]          shadow/BatchNorm_3/FusedBatchNormV3
[ ERROR ]          shadow/BatchNorm_4/FusedBatchNormV3
[ ERROR ]          shadow/BatchNorm_5/FusedBatchNormV3
[ ERROR ]          shadow/BatchNorm_6/FusedBatchNormV3
[ ERROR ]          shadow/BatchNorm_7/FusedBatchNormV3
[ ERROR ]  Part of the nodes was not converted to IR. Stopped.

**Comment by brmarkus** [comment]
Can you describe your environment a little bit, please?
Where did the errors occur? Which tool did you use, how, which command-line parameters? Can you describe your trained model a bit, please?
What do you mean with "export the model to OpenVINO IR", you mean you converted it using the model-optimizer? Which format was it before, TensorFlow?

**Comment by jiazhaozhu** [comment]
The version of openvino is 2019 R3.I solve this issue when I change the version to 2020.1
